numpy/random/mtrand/mtrand.pyx

```diff
@@ -4226,7 +4226,7 @@ cdef class RandomState:
         Returns
         -------
         out : ndarray or scalar
-            Drawn samples from the parameterized hypergeometric distribution.
+            Number of drawn good samples from the parameterized hypergeometric distribution.
```

Comment:
Hmm, I don't think this helps. There are two different samples being described here - `hypergeometric(..., size=N)` returns `N` samples drawn from a hypergeometric distribution.

But the act of _"drawing from a hypergeometric distribution"_ means _"draw samples from (the number of good samples from `nsample` draws without replacement of `[good] * ngood + [bad] * nbad`)"_.

My example is a pretty lousy way to phrase it, but I think it shows what confusion needs to be resolved here.

Reply:
Perhaps instead, keep the original docstring, and add

> Each sample is the number of good items within a randomly selected subset of size `nsample` taken from a set of `ngood` good items and `nbad` bad items.